lib/ansible/module_utils/hwc_utils.py

```diff
@@ -211,6 +226,7 @@ def __init__(self, *args, **kwargs):
             )
         )
 
+        supports_check_mode = True
```

Comment:
It's probably better to add this in the modules themselves, in case you ever want to add a module which can't have check mode (for whatever reason). Also, the way you set this, it will have no effect. You either have to change the line below to `super(HwcModule, self).__init__(*args, supports_check_mode=True, **kwargs)`, or you have to replace this line with `kwargs['supports_check_mode'] = True`, or (my favorite solution) change the module to add this argument when calling `HwcModule.__init__`.

Reply:
Done